views/sign-out.html

```diff
@@ -32,10 +32,11 @@
 
     <form method="post">
 
-      {# Need to have hidden fields for the journey and saveAndResume values so that when the template gets rendered by the validation middleware, #}
-      {#   the journey and saveAndResume values are not lost and passed back into the template for rendering #}
-      <input type="hidden" name="journey" value="{{ journey }}"/>
-      <input type="hidden" name="saveAndResume" value="{{ saveAndResume }}"/>
+      {# Need to have hidden fields for the journey and saveAndResume values so that when the template gets rendered by
+         the validation middleware, #}
```

Comment:
I think the content in line 36 should just be appended on to the end of the previous line, as it was before.

Reply:
Tidied this.